src/blockdigest.cpp

```diff
@@ -16,8 +16,7 @@ std::vector<uint8_t> base_filter::outpoint_data(const COutPoint& outpoint) const
 {
     CDataStream stream(SER_NETWORK, PROTOCOL_VERSION);
     stream << outpoint;
-    std::vector<uint8_t> data(stream.begin(), stream.end());
-    return data;
+    return std::vector<uint8_t>(stream.begin(), stream.end());
```

Comment:
this code still bugs me -- you take the outpoint, you put it into the stream's vector, then you put it into a function-local vector, then you put it into the txdata vector. Can you just serialize directly into the txdata vector?